pkg/repositories/interfaces/schedulable_entity_repo.go

```diff
@@ -0,0 +1,25 @@
+package interfaces
+
+import (
+	"context"
+	"github.com/flyteorg/flyteadmin/pkg/repositories/models"
+)
+
+//go:generate mockery -name=SchedulableEntityRepoInterface -output=../mocks -case=underscore
+
+type SchedulableEntityRepoInterface interface {
```

Comment:
should there be a deactivate here too?

Reply:
Yes i have named it as delete , But deactivate seems right name to use since we are technically just toggling the flag